src/Tools/dotnet-counters/TsvWriter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Diagnostics.Tools.Counters
+{
+    public class TsvWriter : IOutputWriter
+    {
+        Dictionary<string,string> _columnsValue = new Dictionary<string, string>();
+
+        public void Update(string providerName, ICounterPayload payload)
+        {
+            string name = payload.GetName();
+            string value = payload.GetValue();
+
+            if (_columnsValue.TryAdd(name, value))
+            {
+                // setup header file?
```

Comment:
Payload comes in a sequence way, so the output file will have and "increment" form
```
...
0
0	22
0	22	0
0	22	0	1571
0	22	0	1571	1571
0	22	0	1571	1571	1571
...
```
To parse data after shutdown we need:
* header file
* add a "new column" line directly in file, something like
```
col1
0      col2
0	22    col3
0	22	0       col4
0	22	0	1571    col5
0	22	0	1571	1571    col6
0	22	0	1571	1571	1571
...
```